hazelcast/test/src/compact/compact_test_base.h

```diff
@@ -44,6 +44,10 @@ class compact_test_base : public testing::Test
         factory_.reset(
           new HazelcastServerFactory{ "hazelcast/test/resources/compact.xml" });
         member_.reset(new HazelcastServer{ *factory_ });
+
+        if (client != nullptr) {
+            client->shutdown().get();
+        }
```

Comment:
Latest instance will be left, so it is better to add this to destructor or `TearDown`.

Reply:
Fixed